src/main/java/core/basesyntax/Colours.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colours {

```

Comment:
```suggestion
public enum Colour {
```
also enum values should be in upper case

Reply:
what about this comment?
```suggestion
public enum Color {

```